anchore_engine/services/policy_engine/engine/feeds/sync_utils.py

```diff
@@ -0,0 +1,119 @@
+from abc import ABC, abstractmethod
+
+from anchore_engine.db import FeedGroupMetadata
+from anchore_engine.services.policy_engine.engine.feeds.client import (
+    get_feeds_client,
+    get_grype_db_client,
+)
+from anchore_engine.services.policy_engine.engine.feeds.sync import DataFeeds
+from anchore_engine.subsys import logger
+
+GRYPE_DB_FEED_NAME = "grypedb"
```

Comment:
This string also exists on `GrypeDBFeed` as a class variable called __feed_name__. We might want to establish a single source of truth for the name of this feed that we can reference wherever needed as opposed to using static strings in multiple places

Reply:
You are right.

Reply:
Pushed a commit to fix this. I'm still using that constant, `GRYPE_DB_FEED_NAME`, but I'm setting it to the value of the `GrypeDBFeed.__feed_name__`.